ethcore/sync/src/api.rs

```diff
@@ -489,6 +490,7 @@ impl NetworkProtocolHandler for SyncProtocolHandler {
 			io.register_timer(MAINTAIN_SYNC_TIMER, Duration::from_millis(1100)).expect("Error registering sync timer");
 			io.register_timer(CONTINUE_SYNC_TIMER, Duration::from_millis(2500)).expect("Error registering sync timer");
 			io.register_timer(TX_TIMER, Duration::from_millis(1300)).expect("Error registering transactions timer");
+			io.register_timer(DELAYED_PROCESSING_TIMER, Duration::from_millis(2000)).expect("Error registering delayed processing timer");
```

Comment:
```suggestion
			io.register_timer(DELAYED_PROCESSING_TIMER, Duration::from_millis(2100)).expect("Error registering delayed processing timer");
```
The idea here was to use numbers that don't overlap frequently. With 2s, every 5th time we will overlap with `CONTINUE_SYNC_TIMER`.